src/Template/Bake/Form/form.twig

```diff
@@ -0,0 +1,60 @@
+{#
+/**
```

Comment:
Why is array_property.twig without license doc block but this one and others is?
IMO we should still get rid of this noise in all such template twig files as discussed before.